src/core_atmosphere/physics/mpas_atmphys_driver_pbl.F

```diff
@@ -1499,6 +1568,14 @@ subroutine driver_pbl(itimestep,configs,mesh,sfc_input,diag_physics,tend_physics
                   bl_mynn_cloudmix   = bl_mynn_cloudmix    ,                                              &
                   bl_mynn_mixqt      = bl_mynn_mixqt       ,                                              &
                   bl_mynn_ess        = bl_mynn_ess         ,                                              &
+                  mix_chem = mix_chem, &
+                  nchem = num_chem,    &
+                  ndvel = num_chem,    &
+                  chem3d = chem_p,     &
+                  vd3d = ddvel_p,      &
+                  settle3d = tend_chem_settle_p, &
+                  frp_mean = xland_p,  &
+                  emis_ant_no = xland_p, &
```

Comment:
I assume setting them == xland is a temporary hack as opposed to a bug? FRP is important for the enhanced mixing, so this PR will not have that feature?

Reply:
Yes, that is a temporary hack. We will make a quick update of this PR for FRP and enhanced mixing.